setup.py

```diff
@@ -162,6 +175,18 @@ def setup_package():
 
         metadata['version'] = VERSION
     else:
+        if is_numpy_installed() is False:
+            sys.stderr.write("Error: Numerical Python(NumPy) is not installed.\n"
+                             "scikit-learn requires NumPy.\n"
+                             "Instruction installations are available on scikit-learn website: "
+                             "http://scikit-learn.org/stable/install.html\n")
+            sys.exit(1)
```

Comment:
Are you sure this is the correct way to "fail" in setup package? This exits a shell if the user were to `import setup` and call `setup.setup_package`.